scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -73,19 +73,70 @@ def remove_too_many_alt_allele_sites(vds):
 
 
 
+def annotate_adj(
+        mt: hl.MatrixTable,
+        adj_gq: int = 30,
+        adj_ab: float = 0.2,
+) -> hl.MatrixTable:
+    """
+    Annotate genotypes with adj criteria (assumes diploid).
+    Defaults similar to gnomAD values, but GQ >= 20 changed to GQ >= 30 to make up for lack of DP filter.
+    """
+    if "GT" not in mt.entry and "LGT" in mt.entry:
+        print("No GT field found, using LGT instead.")
+        gt_expr = mt.LGT
+    else:
+        gt_expr = mt.GT
+    if "AD" not in mt.entry and "LAD" in mt.entry:
+        print("No AD field found, using LAD instead.")
+        ad_expr = mt.LAD
+    else:
+        ad_expr = mt.AD
```

Comment:
Does this make sense if one is local and one is not? I believe this is being called with both LGT and GT, and only LAD, so this code would use GT and LAD. Also, in case both encodings are available, wouldn't the local encoding be preferable? In particular, the sum of AD would be cheaper to compute for large multi-allelic sites.